### ISSUE TITLE
Certain ordering of task variables in the Editor

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#342]
```
> What steps will reproduce the problem?
1. Open the specification in the Editor
2. Open the Task Decomposition Details of the "Fill_Out_Continuity_Report" task
3. Arrange the task variables in the following order:
generalInfo, producer, director, directorOfPhotography, editor, continuity,
continuityInfo, finalSubmission
4. Save the specification
5. Reopen the specification and the task decomposition details of the above
task

> What is the expected output? What do you see instead?
I would expect to see the task variables in an order as what I specified
and saved in the above, but they are always back to the certain order
everytime when the specification is opened in the Editor.
 
> Please use labels and text to provide additional information.
Editor version: YAWL version 2.0 RC2, build on date 2009.06.15 




```

Original issue reported on code.google.com by `c.ouyang...@gmail.com` on 24 Jun 2009 at 8:40

Attachments:
* [ContinuityReport_Test.yawl](https://storage.googleapis.com/google-code-attachments/yawl/issue-343/comment-0/ContinuityReport_Test.yawl)

**Comment by yawlfoundation** [comment]
Original comment by `c.ouyang...@gmail.com` on 24 Jun 2009 at 11:25
* Changed state: **New**

**Comment by yawlfoundation** [comment]
```
this is fixed in 2.0 final
```

Original comment by `yawl.mic...@gmail.com` on 16 Oct 2009 at 11:59

**Comment by yawlfoundation** [comment]
Original comment by `yawl.mic...@gmail.com` on 16 Oct 2009 at 12:00
* Changed state: **Fixed**
